pkg/client/clientset/versioned/typed/scaling/v1alpha1/scaling_client.go

```diff
@@ -71,7 +70,7 @@ func setConfigDefaults(config *rest.Config) error {
 	gv := v1alpha1.SchemeGroupVersion
 	config.GroupVersion = &gv
 	config.APIPath = "/apis"
-	config.NegotiatedSerializer = serializer.DirectCodecFactory{CodecFactory: scheme.Codecs}
+	config.NegotiatedSerializer = serializer.CodecFactory{}
```

Comment:
this may be incorrect.  When using the scheduled scalar with our local implementation, I get 

```serializer for application/json doesn't exist```

Reply:
this is definitely an issue.  https://github.com/kubernetes/client-go/blob/master/rest/request.go#L1218
r.decoder is nil...

Reply:
try:

* https://pkg.go.dev/k8s.io/apimachinery/pkg/runtime#NewScheme
* https://pkg.go.dev/k8s.io/apimachinery/pkg/runtime/serializer#NewCodecFactory